lib/board.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './cell.rb'
+require_relative './ui.rb'
+
+class Board
+  include UI
+  attr_accessor :values
+  def initialize
+    @values = Array.new(3) { |row| Array.new(3) { |col| Cell.new(row * 3 + col + 1) } }
```

Comment:
It's ok to call Array.new here (since Ruby's built in classes won't change anytime soon) but instantiating your own created class inside another class isn't a good practice.